src/medCore/gui/toolboxes/medActionsToolBox.cpp

```diff
@@ -55,6 +56,17 @@ medActionsToolBox::medActionsToolBox( QWidget *parent /*= 0*/, bool FILE_SYSTEM
     d->buttonsWidget = new QWidget(this);
     d->noButtonsSelectedWidget = new QWidget(this);
 
+    // Information Widget for File System tab
+    d->informationWidget = new QWidget(this);
+    QLabel* informationLabel = new QLabel(tr("To import DICOMs, select the directory containing these files.\nDo not select them separately."),
+                                          d->informationWidget);
+    informationLabel->setObjectName("actionToolBoxLabel");
+    informationLabel->setSizePolicy(QSizePolicy::Ignored, QSizePolicy::Preferred);
+    informationLabel->setWordWrap(true);
+    QHBoxLayout* informationLayout = new QHBoxLayout(d->informationWidget);
+    informationLayout->addWidget(informationLabel, 0, Qt::AlignLeft);
+    this->addWidget(d->informationWidget);
```

Comment:
Can't the label widget be added directly instead of encompassing it in another widget?

Reply:
At the bottom of that function is the `d->noButtonsSelectedWidget` label/widget which use this intermediate layout too. It allows to Align the widget where you want, to position the widget. Furthermore, i tried without that and the widget was not seen.

Reply:
Ok then.